napari/_vispy/vispy_points_layer.py

```diff
@@ -1,10 +1,11 @@
 import numpy as np
-from vispy.scene.visuals import Line, Compound
+from vispy.scene.visuals import Line, Compound, Text
 from .markers import Markers
 from vispy.visuals.transforms import ChainTransform
 
 from .vispy_base_layer import VispyBaseLayer
 from ..utils.colormaps.standardize_color import transform_color
+from .text_utils import update_text
```

Comment:
I love this approach and how clean `vispy_points_layer.py` and `vispy_shapes_layer.py` is!!!